CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ Released: TBD
   from the `options.grammarSource` property. That property can contain arbitrary
   data,for example, path to the currently parsed file.
   [@Mingun](https://github.com/peggyjs/peggy/pull/95)
+- Parsers now can use two new functions in actions/predicates: `offset` that returns
+  just the start offset, and a `range` that returns the array `[start, end]` offsets.
```

Comment:
I know we have this issue in the existing docs, but `offset` is in 16 bit code units (js characters) for the moment, right?  If we parse by codepoint, this will be in codepoints, I think?  I think we should also make more clear that `range` is `[startOffset, endOffset]`.

Reply:
Yes, it is in [`charAt`]/[`charCodeAt`] units and yes, should be in codepoints if we parse by codepoint.

>  I think we should also make more clear that `range` is `[startOffset, endOffset]`.
> ...the array `[start, end]` offsets.

Is that not enough? From my perspective, it is, but if you can formulate better, then welcome.

[`charAt`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt
[`charCodeAt`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt

Reply:
Let's make the units more clear for offset in #148, then I think this issue goes away.